Resources/Prototypes/_NF/Entities/Structures/Wallmounts/Signs/posters.yml

```
@@ -49,6 +49,26 @@
     sprite: _NF/Structures/Wallmounts/posters.rsi
     state: nf_poster5_legit
 
+- type: entity
+  parent: PosterBase
+  id: NFPosterLegitGunsPlease
+  name: "Guns Please!"
+  description: An adorable mothgirl in a cute hat asks that you please do your civic duty and give your guns to NFSD staff.
+  components:
+  - type: Sprite
+    sprite: _NF/Structures/Wallmounts/posters.rsi
+    state: nf_poster6_legit
+
+- type: entity
+  parent: PosterBase
+  id: NFPosterLegitHaveYouSeenThisEgg
+  name: "Have You Seen This Egg"
+  description: Please immediately inform NFSD staff of any strange sightings like roaming space monsters, gigantic purple eggs, and strange notes left behind posters or vending machines.
```

```suggestion
  description: Please inform NFSD personnel of any strange sightings like roaming space monsters, gigantic purple eggs, and strange notes left behind posters or vending machines.
```
Immediately and please set an odd tone together, I'd pick one or the other.

Staff is an odd one - "the NFSD", "NFSD personnel", or "NFSD officers"/"an NFSD officer" would work depending on the tone you want to set.